decompiler/util/default.json

```diff
@@ -291,6 +291,48 @@
       "is_hidden_from_gui": false
     }
   },
+  "variable-name-generation": {
+    "notation": {
+      "default": "default",
+      "title": "Variable Naming Convention",
+      "type": "string",
+      "enum": ["default", "system_hungarian"],
+      "enumDescriptions": [
+        "Default setting (e.g. int var_0)",
+        "System Hungarian (e.g. int iVar0; float fVar0)"
+      ],
+      "description": "Selects the naming convention for variable names.",
+      "is_advanced_setting": false
+    },
+    "variable_name": {
+      "default": "Var",
+      "title": "Variable Base Name",
+      "type": "string",
+      "description": "",
+      "is_advanced_setting": false
```

Comment:
might be vonfusing that "Variable base Name" only applies to hungarian notation, or did I miss something?

Reply:
Changed the description. 
Maybe we could make some options more generic, so that the default naming convention could be changed as well?

Reply:
of course, nothing set in stone here.